CI/PSI/Jenkinsfile

```diff
@@ -36,18 +36,18 @@ podTemplate(label: 'docker',
      // also available as a Groovy variable—note double quotes for string interpolation
      echo "$USERNAME"
      echo "$PASSWORD"
-     sh "git clone https://$USERNAME:$PASSWORD@git.psi.ch/MELANIE/catamel-psisecrets.git"
-     sh "git clone https://$USERNAME:$PASSWORD@git.psi.ch/MELANIE/catamel-psiconfig.git"
+     sh "git clone https://$USERNAME:$PASSWORD@git.psi.ch/MELANIE/scicat-backend-psisecrets.git"
```

Comment:
@stephan271 I think we can get rid of this file. Do you agree?

Reply:
Yes.